push-notifications/ios/Plugin/PushNotificationsPlugin.swift

```diff
@@ -0,0 +1,188 @@
+import Foundation
+import Capacitor
+import FirebaseCore
+import FirebaseMessaging
+import UserNotifications
+
+enum PushNotificationError: Error {
+    case tokenParsingFailed
+    case tokenRegistrationFailed
+}
+
+enum LocationPermissions: String {
```

Comment:
Should probably be `PushNotificationsPermissions`

Reply:
Good catch!